mlrun/artifacts/manager.py

```diff
@@ -325,7 +325,7 @@ def log_artifact(
 
     def update_artifact(self, producer, item: Artifact):
         self.artifact_uris[item.key] = item.uri
-        self._log_to_db(item.db_key, producer.project, producer.inputs, item)
+        self._log_to_db(item.db_key, producer.name, None, item)
```

Comment:
This doesn't seem right. It will only work properly if the producer is a project, otherwise it will use the run-name as the project-name which is wrong.

Reply:
will fix